requirements.txt

```diff
@@ -1,11 +1,9 @@
 #
-# This file is autogenerated by pip-compile with python 3.7
-# To update, run:
+# This file is autogenerated by pip-compile with Python 3.7
+# by the following command:
 #
-#    pip-compile --index-url=https://nexus.osdc.io/repository/pypi-all/simple
+#    pip-compile
```

Comment:
Did you rerun the pip-compile to pull in all new libraries?

Reply:
I rerun pip-compile, and it just removed the `--index-url=https://nexus.osdc.io/repository/pypi-all/simple` the rest are the same. 
I removed this index-url to keep travis work.